Sources/SIL/SIL.swift

```diff
@@ -10,6 +10,11 @@ public class Module {
         let parser = try SILParser(forPath: silPath)
         return try parser.parseModule()
     }
+
+    public static func parse(fromString silString: String) throws -> Module {
```

Comment:
Per Swift API Design Guidelines, argument labels should not repeat the type information.
```suggestions
public static func parse(from silString: String) throws -> Module {
```